pkg/common/onedir/_salt_onedir_extras.py

```diff
@@ -0,0 +1,16 @@
+import pathlib
+import sys
+
+# Discover the extras-<py-major>.<py-minor> directory
+extras_parent_path = pathlib.Path(__file__).parent.parent
```

Comment:
Is this supposed to be `parent.parent.parent` like before when we did the oneliner?